README.md

```diff
@@ -4,16 +4,33 @@
 ![](./docs/pics/arana-logo.png)
 
 ## Introduction | [中文](https://github.com/dubbogo/arana/blob/master/docs/README_ZH.md)
-hello, db.
+
+Arana is a db proxy. It can be deployed as a sidecar, also can be deployed standalone.
```

Comment:
Arana is a db proxy. It can be deployed as a sidecar, also can be deployed standalone as a proxy.